components/verifalia/actions/verify-email/verify-email.mjs

```diff
@@ -30,7 +30,7 @@ export default {
       description: "The higher the quality level the longer it could take to complete the verification; by default, " +
                 "we use the configured default quality level for your Verifalia user.",
```

Comment:
```suggestion
      description: "The higher the quality level the longer it could take to complete the verification; by default, we use the configured default quality level for your Verifalia user.",
```